Solidus-Swift-SDK/EndpointsBase.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  EndpointsBase.swift
+//  Solidus-Swift-SDK
+//
+//  Created by Francisco Flores on 5/3/16.
+//  Copyright © 2016 Omar Guzman. All rights reserved.
+//
+
+import Foundation
+
+struct EndpointsBase {
+    static var products = "/api/products"
```

Comment:
ok, so, if solidus engine is mounted in a namespace, you will use `www.store.com/spree` ?